redpanda/Chart.yaml

```
@@ -21,7 +21,7 @@ maintainers:
     url: https://github.com/orgs/redpanda-data/people
 type: application
 version: 2.0.0-alpha.0
-appVersion: 21.11.16
+appVersion: 22.1.4
```

```suggestion
appVersion: 22.1.5
```
This is a moving target, but as of now `22.1.5` is the latest.

---

resolved [46f6b62](https://github.com/redpanda-data/helm-charts/pull/117/commits/46f6b628b22162234c01501ab2e222b359aebc5e)